### DISCUSSION TITLE
TTF Font renders white squares on Android

### DISCUSSION THREAD

**Discussion by AlexandreK38** [axmolengine/axmol#1944] [Q&A] [answered]
Hi!
I switched from Cocos2d 4 to Axmol 2.1.2 few days ago.
Today I switched from 2.1.2 to 2.1.3 but somehow my fonts using TTF don't work anymore on Android anymore, it renders white squares. The font file is found; note that on iOS it is rendering as expected.
Any thoughts about what code change since 2.1.2 could cause this issue?

**Comment by AlexandreK38** [comment, marked as answer]
I don't know why but I needed to to "Clean Build", build from zero again and it worked.

**Reply by GamesStudioTwo** [reply]
I Switched 2 weeks ago so happy and Welcome by the way.
